docs/operate/customize/require_community.md

```diff
@@ -0,0 +1,114 @@
+# Require Community for Record Publication
+
+## Overview
+
+Institutional policies may require all published records to belong to at least one community. InvenioRDM enforces this requirement through the RDM_RECORD_ALWAYS_IN_COMMUNITY configuration parameter.
+
+## Configuration
+Enable mandatory community assignment in `invenio.cfg`:
+
+```python
+# Require community membership for all published records
+RDM_RECORD_ALWAYS_IN_COMMUNITY = True
+```
+Default State: Disabled `False`
+
+## Enforcement Behavior
+
+When enabled:
+
+- Records require at least one community for publication
+    ![Remove last community](../customize/imgs/cannot_publish_without_community.png)
+
+- Users cannot remove the last community from a draft
+    ![Record require at least one community for publication](../customize/imgs/rec_should_be_part_at_leat_one_comm.png)
+
+- Only privileged users (default: super administrators) can bypass this requirement
+
+- Unauthorized publication attempts without communities fail with explicit error messages
```

Comment:
Is this point the same as the one on line 20 or different? (if different, more details? or can both be merged?)